sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -62,6 +68,10 @@
 
         this.serviceVersion = serviceVersion;
 
+        // Is null by default. If not null, then the user provided a custom json serializer for the convenience layer to use.
+        // If null, then mapper will be used instead. See DeserializationHelpers for more details
```

Comment:
nit: can we add a link to `DeserializationHelpers` here?

Reply:
Links only work in javadocs, not comments

Reply:
:facepalm:, my bad!